CONTRIBUTING.rst

```diff
@@ -395,8 +381,76 @@ exists, be sure to add the resulting image with
 
     git add tests/plotting/image_cache/*
 
+Building the Documentation
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+Build the documentation on Linux or Mac OS with:
+
+.. code:: bash
+
+   make -C doc html
+
+Build the documentation on Windows with:
+
+   cd doc
+   python -msphinx -M html . _build
+
+The generated documentation can be found in the ``doc/_build/html``
+directory.
+
+The first time you build the documentation locally will take a while as all the
+examples need to be built. After the first build, the documentation should take
+a fraction of the time.
+
+Clearing the local build
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you need to clear the locally built documentation, run:
+
+.. code:: bash
+
+   make -C doc clean
+
+This will clear out everything, including the examples gallery. If you only
+want to clear everything except the gallery examples, run:
+
+.. code:: bash
+
+   make -C doc clean-except-examples
+
+This will clear out the cache without forcing you to rebuild all the examples.
+
+Full documentation build
+^^^^^^^^^^^^^^^^^^^^^^^^
+By default, the documentation build places class documentation on single pages
+rather than generating a table and placing class methods and attributes on
+individual pages. Figures are also sized to ``(400, 300)`` rather than the
+default ``(1024, 768)``. This is to minimize the time it takes to build
+documentation locally. If you wish to generate the full documentation, enable
+it with::
+
+.. code:: bash
+
+   export FULL_DOC_BUILD=TRUE
+   make -C doc html
+
+This will generate the same documentation as rendered online, but as each
+individual class method and attribute will have its own documentation page, the
+documentation will take much longer to build.
+
+Parallel Documentation Build
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+You can improve your documentation build time on Linux and Mac OS with:
+
+.. code:: bash
+
+   make -C doc html
```

Comment:
phtml?

Reply:
I figured that with the mention of `phtml` below that it's up to the user if they want to spin up all their cores, but I won't advocate it as the default since some people might want to run it in the background and now the vast majority of the time will be in the single threaded examples.

Reply:
I guess I don't follow this section then.  The code here is exactly the same as the first code block under `Building the documentation`.  Is there a flag missing from this example?  I don't see `phtml` mentioned in this `Building the documentation` section.

Reply:
Sorry, should have actually referenced the file before commenting. Yes, it should be phtml.

```suggestion
   make -C doc phtml
```